src/test/fuzz/addrman.cpp

```diff
@@ -101,7 +108,7 @@ class CAddrManDeterministic : public CAddrMan
                 if (n > 0 && mapInfo.size() % n == 0 && mapAddr.find(addr) == mapAddr.end()) {
                     // Add to the "tried" table (if the bucket slot is free).
                     const CAddrInfo dummy{addr, source};
-                    const int bucket = dummy.GetTriedBucket(nKey, m_asmap);
+                    const int bucket = dummy.GetTriedBucket(nKey, *g_setup->m_node.netgroupman);
```

Comment:
I think this should use `m_netgroupman` which may have been fuzzed prior to calling `Fill()`, instead of the empty  `g_setup->m_node.netgroupman`, see CI fuzz test fail.

Reply:
You're absolutely right, of course. Now fixed.